internal/builder/lifecycle.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	DefaultLifecycleVersion    = "0.7.4"
+	DefaultLifecycleVersion    = "0.7.5"
```

Comment:
I think it should be rolled back to 0.7.4.

Reply:
See other similar comment about Default Lifecycle Version.